code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -64,19 +71,42 @@
 	move_force = MOVE_FORCE_DEFAULT
 	move_resist = MOVE_RESIST_DEFAULT
 	pull_force = PULL_FORCE_DEFAULT
-	..(gibbed)
+	..()
+
+/mob/living/simple_animal/hostile/asteroid/goliath/gib()
+	if(prob(1))//goliaths eat rocks and thus have a tiny chance to contain a number of gems
+	        var/obj/item/gem/to_drop = pick(/obj/item/gem/rupee, /obj/item/gem/fdiamond, /obj/item/gem/void, /obj/item/gem/phoron)
```

Comment:
```suggestion
		var/obj/item/gem/to_drop = pick(/obj/item/gem/rupee, /obj/item/gem/fdiamond, /obj/item/gem/void, /obj/item/gem/phoron)
```
This line has spaces instead of a tab at this place